cmd/explaintest/r/partition_pruning.result

```diff
@@ -4273,3 +4273,15 @@ Union_9	750.00	root
   └─Selection_17	250.00	cop	ge(test.t1.a, 10), le(test.t1.a, 43)
     └─TableScan_16	10000.00	cop	table:t1, partition:p2, range:[-inf,+inf], keep order:false, stats:pseudo
 drop table t0, t1;
+drop table if exists t;
+create table t(a timestamp) partition by range(unix_timestamp(a)) (partition p0 values less than(unix_timestamp('2019-02-16 14:20:00')), partition p1 values less than (maxvalue));
+insert into t values('2019-02-16 14:19:59'), ('2019-02-16 14:20:01');
+explain select * from t where a between time'14:19:00' and timestamp'2019-02-16 14:21:00';
```

Comment:
This SelectStmt will panic during executing?
Do we have an issue to record this bug?

Reply:
It won't panic, but get a wrong result though the execution plan is correct after this PR. It was recorded in this issue https://github.com/pingcap/tidb/issues/9325